lib/src/buttons/push_button.dart

```diff
@@ -202,6 +231,10 @@ class PushButton extends StatefulWidget {
   /// Can still be overridden if the [color] attribute is non-null.
   final bool? secondary;
 
+  /// An optional iconData that can be added to a button.
+  /// This is an iconData instead of icon so we can control the size
```

Comment:
```suggestion
  /// An optional `IconData` that can be added to a button.
  /// 
  /// This is an `IconData` instead of an `Icon` so that the icon's size can be controlled internally.
```